### DISCUSSION TITLE
Scaling Ethereum 2024 hackathon winners, builder bounties, and a new project showcase page

### DISCUSSION THREAD

**Discussion by oceans404** [NillionNetwork/discussions#30] [Announcements]
## 🏆 Celebrating the Nillion Scaling Ethereum hackathon winners

We're thrilled to announce the winners of the Scaling Ethereum 2024 hackathon by [EthGlobal](https://ethglobal.com/). During the event, we introduced the [developer preview of the Nillion SDK](https://docs.nillion.com/nillion-sdk-and-tools), featuring Python and JavaScript clients and the new Nada language. 7 innovative teams leveraged Nillion to build unique projects that are now available for everyone to view on our Showcase page.

A huge thank you to all the participants for their hard work and creativity. Special congratulations to the three winning projects, awarded 5,000 USDC:

#### :desktop_computer: Best novel use cases of blind computation with Nillion
 - [🧬 Monadic DNA](https://ethglobal.com/showcase/monadic-dna-h7k7q)
 - [⛴️ BattleshipX](https://ethglobal.com/showcase/battleshipx-0gc9h)

#### :globe_with_meridians: Best application integrating Nillion’s tech
 - [🔒 PrivyCal](https://ethglobal.com/showcase/privycal-i1gub)

## 💸 Announcing builder bounties

We're always looking for ways to reward developers for building on and with us. Today we're launching the 1st of many Nillion builder bounties on [Bountycaster](https://www.bountycaster.xyz/):  Earn 500 USDC for [building a Hangman game on Nillion](https://warpcast.com/~/conversations/0x951ccaa7fa6ae3c5f34998ab951c1c28bea3b8d3)

## 🌟 Showcase Page

What have people been building on Nillion? Has anyone built a game or something AI related? Can we see their code?

Introducing the [Nillion developer showcase page](https://docs.nillion.com/showcase), a list of awesome open source projects built on Nillion by YOU, our dev community. Add your project to the Showcase page with a PR. If you have a question about a specific community project, you can ask the team/builder directly in the linked Discussions.

<img width="1470" alt="Screenshot 2024-04-26 at 1 49 41 PM" src="https://github.com/NillionNetwork/discussions/assets/91382964/d6c4c83f-ba74-4eaa-a275-96af681547fe">

**Comment by wwwehr** [comment]
So impressed with all the entries. Amazing work!
